docs/workshop/lesson2.md

```diff
@@ -90,7 +90,7 @@ Before you start creating your assessments, make sure you have at least 4 questi
 - Use `"mode": "Public"`
 - Use `"type":"Homework"`
 - Questions appear in randomized order
-- Homework submitted at the deadline receive 100% credit. Homework submitted up to 4 days late receive 70% credit. Homework submitted at least 2 days before the deadline get 5% bonus.
+- Homework submitted by the deadline receives 100% credit. Homework submitted up to 4 days late is capped at 70% credit. Homework submitted at least 2 days before the deadline get 5% bonus if they receive full credit. Learn more about credit in the [access control docs](https://prairielearn.readthedocs.io/en/latest/accessControl/#credit).
```

Comment:
I think it's important to clarify that the full credit must be achieved before the 2 days early scenario. The proposed phrasing leaves open the possibility of bonus if completion happens later. Also, Homework is "it", not "they".

```suggestion
- Homework submitted by the deadline receives 100% credit. Homework submitted up to 4 days late is capped at 70% credit. Homework submitted early gets 5% bonus if it receives full credit at least 2 days before the deadline. Learn more about credit in the [access control docs](https://prairielearn.readthedocs.io/en/latest/accessControl/#credit).
```

Reply:
Thanks, that grammar was all over the place to start with and I managed to make it worse 😂 in my mind I think "they" was "the students submitting the homework".

Here's an iteration on yours (IMO it is in fact better to talk about students getting credit than "the homework" getting credit):

```suggestion
- Students submitting by the deadline receive 100% credit. Students submitting up to 4 days late are capped at 70% credit. Students get a 5% bonus if they receive full credit at least 2 days before the deadline. Learn more about credit in the [access control docs](https://prairielearn.readthedocs.io/en/latest/accessControl/#credit).
```

Reply:
I think that's good, though "if they receive full credit" is actually overloading the term "credit" to mean something else here. Can we use a different term? Maybe "full score"? Won't block on that.

Reply:
"...receive a 100% score" maybe?

Reply:
I changed my suggestion above to the following:

> If students earn 100% at least 2 days before the deadline, they'll receive an additional 5% bonus for a total of 105%.

I'll land this.